### ISSUE TITLE
 Display error

### ISSUE THREAD

**Issue by lexazxc** [surfgg/dragon-knight#2]
If several users play at the same time, then the statistics (which is on the right) will be the last logged-in user, and also happens with hidden links (for example, the admin panel)

**Comment by splashsky** [comment]
Yes, this is an issue I've been observing with recent test runs. I'm not fully sure what the cause is, but reworking the auth system into something PHP Session-based is currently top priority to address this problem. Right now it's pure Cookies and it's complete butt.
